### PR TITLE
Ziad/mint profile staking last line

### PR DESCRIPTION
**Pull request by towardsnewtech**

### CODE REVIEW

File: components/MintPage/MintDay/MintBoard/QuantityCards.tsx
Comment (sweetmantech) [inline components/MintPage/MintDay/MintBoard/QuantityCards.tsx:1]: 💯 nice component ✨

File: hooks/useCheckNetwork.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      return false
+    }
+
+    return true
+  }
+
+  return { checkNetwork }
+}
+
+export default useCheckNetwork
```
Comment (sweetmantech) [inline hooks/useCheckNetwork.ts:27]: perfect hook 🪝

**Review by SweetmanTech — APPROVED** [review]
ship it 🚢

**Review by SweetmanTech — APPROVED** [review]
ship it 🚢

**Pull request merged**